codegen-server-test/python/model/pokemon.smithy

```diff
@@ -41,3 +45,40 @@ structure StreamPokemonRadioOutput {
 
 @streaming
 blob StreamingBlob
+
+@readonly
+@http(uri: "/union", method: "POST")
+operation GetUnion {
+    input: GetUnionInput,
+    output: GetUnionOutput,
+    errors: [ValidationException]
+}
+
+@input
+structure GetUnionInput {
+    @required
+    data: MyUnion
+}
+@output
+structure GetUnionOutput {
+    @required
+    data: MyUnion
+}
+
+structure ConstrainedStuff {
+    //@length(min: 1, max: 100)
+    inner: String
+}
+
+union InnerUnion {
+    a: Integer
+    b: ConstrainedStuff
+}
+
+union MyUnion {
```

Comment:
Should we move those types to an end-to-end test? We will probably get rid of Python specific Pokémon model when we support event streaming tho, so it is also fine to keep them here for now.

Reply:
I am really thinking to just get rid of this piece of model until we have an operation with union that makes sense. We have unions covered by `misc` and `ebs` right now. Thoughts?

Reply:
Yeah, that makes sense. We'll have unions with event streaming, so I'm fine to remove them from Pokémon service for now